### ISSUE TITLE
Possibility to exclude containers with a label?

### ISSUE THREAD

**Issue by saschabrockel** [willfarrell/docker-autoheal#126]
Is it possible to use a label to specify containers that should be excluded from autoheal instead of being included? I have 100 containers and I don't want to add labels for inclusion to 99 ones because one shouldn't be auto-healed.

**Comment by tmknight** [comment]
This seems to be the same as #73 which has been addressed by #84

**Comment by MrClever** [comment]
This is as good of an explanation as any other on how to achieve what you're asking: https://sdr-enthusiasts.gitbook.io/ads-b/useful-extras/auto-restart-unhealthy-containers#only-monitor-and-restart-a-specific-set-of-containers

**Comment by saschabrockel** [comment]
@MrClever as I stated, it's not nice to change 99 containers for that and add a label to all of them.

**Comment by MrClever** [comment]
> MrClever as I stated, it's not nice to change 99 containers for that and add a label to all of them.

@saschabrockel Fair enough - I forget not everyone auto-generates their configuration files from templates etc 👍🏻 In my case, it was literally two lines to update hundreds of images.

**Comment by saschabrockel** [comment]
@MrClever what kind of templates? I mean we all have compose files I guess but that's obviously not what you mean.

**Comment by MrClever** [comment]
I have a bespoke system for generating compose files etc which populates all the repeated stuff. The container configurations are stored in a custom MySQL DB...it wasn't my creation, but it "works" (*most of the time*). These days, I'd prefer to replace it with a "proper" orchestration framework, but it works well enough that I can't really justify the time/cost to replace it :(

**Comment by saschabrockel** [comment]
Okay never heard of it. I know infrastructure as code and things like Ansible of course but never heard of something that is stored in MySQL 😆

**Comment by MrClever** [comment]
> Okay never heard of it. I know infrastructure as code and things like Ansible of course but never heard of something that is stored in MySQL 😆

Yeh, like I said, give me Ansible/Puppet/Chef/etc any day...this thing can be a pain, but it also makes mass changes to configurations easy.
